packages/ses/docs/lockdown.md

```diff
@@ -405,12 +401,7 @@ handler will only be triggered by unhandled rejections when they are finalized.
 This enables the program to attach rejection handlers asynchronously without
 triggering the SES trap handler.
 
-- `'platform'`: same as `'report'` for web and Node.js
 - `'report'`: just report finalized unhandled rejections to the tamed console so stack traces appear.
-- `'exit'`: reports and exits on Node.js, reports and navigates away on the
-  web.
-- `'abort'`: reports and aborts a Node.js process, leaving a core dump for
-  postmortem analysis, reports and navigates away on the web.
 - `'none'`: do not install traps for finalized unhandled rejections. Errors are
   likely to appear as `{}` when they are reported by the default trap.
```

Comment:
> Errors are likely to appear as `{}` when they are reported by the default trap.

Uh oh. I am confused about the meaning of `'none'`. What default trap? The platform's original behavior? In that case it should be called `'platform'` not `'none'`. I thought `'none'` meant to suppress any reporting.

Reply:
I used the same meaning as `errorTrapping`: `'none'` means don't install a handler.

Reply:
I agree with @michaelfig’s interpretation of `'none'`: do nothing, do not install handlers.

Reply:
Huh. I see that you're right, and it even uses the same text as its explanation. But `errorTrapping` also has a `'platform'` option. If `'none'` means "leave the platform behavior as is", what does `'platform'` mean? I cannot figure it out from the text in lockdown.md.

Maybe a question for @kriskowal ?

Reply:
For `errorTrapping`, `'platform'` means to use the idiomatic behavior for the platform: either `'exit'` in Node.js or `'report'` on the web. The other options override the platform-specific idiom, forcing a non-idiomatic behavior if elected. For example, `'exit'` on the web navigates away rather than allow the program to continue running, rendering the error undiscoverable.

Reply:
``platform`` means to install a handler that has different behaviour depending on the platform.

Reply:
I still do not understand. Operationally, what is the observable difference between `'platform'` and `'none'`? Which is closer to what the raw platform itself does?

Reply:
> I still do not understand. Operationally, what is the observable difference between `'platform'`

Print the deep stack trace.

> and `'none'`?

Don't print the deep stack trace.

Reply:
Thanks!